src/app/(nextjs_migration)/(guest)/breach-details/[breachName]/page.tsx

```diff
@@ -0,0 +1,368 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image, { StaticImageData } from "next/image";
+import BreachDetailScanImage from "../../../../../client/images/breach-detail-scan.svg";
+import "../../../../../client/css/partials/breachDetail.css";
+import { getL10n } from "../../../../functions/server/l10n";
+import { getBreachByName } from "../../../../../utils/hibp";
+import {
+  getAllPriorityDataClasses,
+  getAllGenericRecommendations,
+} from "../../../../../utils/recommendations";
+import { BreachLogo } from "../../../../components/server/BreachLogo";
+import {
+  getBreachIcons,
+  getBreaches,
+} from "../../../../functions/server/getBreaches";
+import { Breach } from "../../../../transitionTypes";
+import { getLocale } from "../../../../functions/server/l10n";
+
+import Glyph1 from "../../../../../client/images/social-security-numbers.svg";
+import Glyph3 from "../../../../../client/images/passwords.svg";
+import Glyph4 from "../../../../../client/images/bank-account-numbers.svg";
+import Glyph5 from "../../../../../client/images/credit-cards.svg";
+import Glyph6 from "../../../../../client/images/credit-card-cvvs.svg";
+import Glyph7 from "../../../../../client/images/partial-credit-card-data.svg";
+import Glyph8 from "../../../../../client/images/ip-addresses.svg";
+import Glyph9 from "../../../../../client/images/historical-passwords.svg";
+import Glyph10 from "../../../../../client/images/security-questions-and-answers.svg";
+import Glyph11 from "../../../../../client/images/phone-numbers.svg";
+import Glyph12 from "../../../../../client/images/email-addresses.svg";
+import Glyph13 from "../../../../../client/images/dates-of-birth.svg";
+import Glyph14 from "../../../../../client/images/pins.svg";
+import Glyph15 from "../../../../../client/images/physical-addresses.svg";
+import MoreIcon from "../../../../../client/images/more.svg";
+
+const glyphs: Record<string, StaticImageData> = {
+  "social-security-numbers": Glyph1,
+  passwords: Glyph3,
+  "bank-account-numbers": Glyph4,
+  "credit-cards": Glyph5,
+  "credit-card-cvvs": Glyph6,
+  "partial-credit-card-data": Glyph7,
+  "ip-addresses": Glyph8,
+  "historical-passwords": Glyph9,
+  "security-questions-and-answers": Glyph10,
+  "phone-numbers": Glyph11,
+  "email-addresses": Glyph12,
+  "dates-of-birth": Glyph13,
+  pins: Glyph14,
+  "physical-addresses": Glyph15,
+};
```

Comment:
Suggested from [above comment](https://github.com/mozilla/blurts-server/pull/3049/files#r1196883793): 

I converted everything to camelCase with a leading `glyph`. Hopefully this helps? 

```suggestion
import glyphSocialSecurityNumbers from "../../../../../client/images/social-security-numbers.svg";
import glyphPasswords from "../../../../../client/images/passwords.svg";
import glyphBankAccountNumbers from "../../../../../client/images/bank-account-numbers.svg";
import glyphCreditCards from "../../../../../client/images/credit-cards.svg";
import glyphCreditCardCvvs from "../../../../../client/images/credit-card-cvvs.svg";
import glyphPartialCreditCardData from "../../../../../client/images/partial-credit-card-data.svg";
import glyphIpAddresses from "../../../../../client/images/ip-addresses.svg";
import glyphHistoricalPasswords from "../../../../../client/images/historical-passwords.svg";
import glyphSecurityQuestionsAndAnswers from "../../../../../client/images/security-questions-and-answers.svg";
import glyphPhoneNumbers from "../../../../../client/images/phone-numbers.svg";
import glyphEmailAddresses from "../../../../../client/images/email-addresses.svg";
import glyphDatesOfBirth from "../../../../../client/images/dates-of-birth.svg";
import glyphPins from "../../../../../client/images/pins.svg";
import glyphPhysicalAddresses from "../../../../../client/images/physical-addresses.svg";
import MoreIcon from "../../../../../client/images/more.svg";

const glyphs: Record<string, StaticImageData> = {
  "social-security-numbers": glyphSocialSecurityNumbers,
  passwords: glyphPasswords,
  "bank-account-numbers": glyphBankAccountNumbers,
  "credit-cards": glyphCreditCards,
  "credit-card-cvvs": glyphCreditCardCvvs,
  "partial-credit-card-data": glyphPartialCreditCardData,
  "ip-addresses": glyphIpAddresses,
  "historical-passwords": glyphHistoricalPasswords,
  "security-questions-and-answers": glyphSecurityQuestionsAndAnswers,
  "phone-numbers": glyphPhoneNumbers,
  "email-addresses": glyphEmailAddresses,
  "dates-of-birth": glyphDatesOfBirth,
  pins: glyphPins,
  "physical-addresses": glyphPhysicalAddresses
};
```

Reply:
nit: re: @maxxcrawford's comment/suggestion above, change from "MoreIcon" to "glyphMore" for consistency?

```diff
-import MoreIcon from "../../../../../client/images/more.svg";
+import glyphMore from "../../../../../client/images/more.svg";
```

Reply:
Aaargh, of course I only saw this after having manually done the rename :facepalm: 

Although I'm OK with camelCase instead of PasacalCase: ebd583eeb. Also, I just realised I don't even really know what "glyph" means.